qa/tasks/cephfs/fuse_mount.py

```diff
@@ -321,7 +321,7 @@ def umount(self, cleanup=True):
             log.info('Running fusermount -u on {name}...'.format(name=self.client_remote.name))
             stderr = StringIO()
             self.client_remote.run(
-                args=['sudo', 'fusermount', '-u', self.hostfs_mntpt],
+                args=['sudo', 'timeout', '300', 'fusermount', '-u', self.hostfs_mntpt],
                 stderr=stderr, timeout=UMOUNT_TIMEOUT, omit_sudo=False)
```

Comment:
Both the FUSE umount and the kernel client umount already have `timeout=UMOUNT_TIMEOUT` passed to `run` (also set to 300).  Is that somehow ignored?

Reply:
paramiko ssh command timeouts don't really work the way you would think. I don't know exactly why, but the timeout just shuts down the pipe but doesn't actually cause the command to return (especially if it's in an uninterruptible state).

Reply:
It would be good to note this in the commit message which is currently empty.

Reply:
good point, done